core/unit_tests/test_exceptions.py

```diff
@@ -46,6 +46,13 @@ def test_ctor_explicit(self):
         self.assertEqual(e.message, 'Testing')
         self.assertEqual(list(e.errors), [ERROR])
 
+    def test_rpc_only_errors_not_exported(self):
+        from google.cloud.exceptions import GoogleCloudRPCError
+        from google.cloud.exceptions import _HTTP_CODE_TO_EXCEPTION
+        http_exceptions = _HTTP_CODE_TO_EXCEPTION.values()
```

Comment:
I know the other examples don't have this but could you add a space between the imports and the test code?